test/form/index.js

```diff
@@ -6,7 +6,7 @@ const { extend, loadConfig, normaliseOutput } = require('../utils.js');
 
 const samples = path.resolve(__dirname, 'samples');
 
-const FORMATS = ['amd', 'cjs', 'system', 'es', 'iife', 'umd'];
+const FORMATS = ['amd', 'cjs', 'system', 'es', 'esm', 'iife', 'umd'];
```

Comment:
Did you mean to include both `es` and `esm` here or just use `esm`?

Reply:
Both. Esm is an alias which we also should test

Reply:
Yes, but it will slow down the tests if we're rerunning each and every es module format test twice now. Perhaps just one test is necessary.